cuecfg/toml.go

```diff
@@ -0,0 +1,19 @@
+// Copyright 2022 Jetpack Technologies Inc and contributors. All rights reserved.
+// Use of this source code is governed by the license in the LICENSE file.
+
+package cuecfg
+
+import (
+	"github.com/pelletier/go-toml/v2"
+)
+
+// TODO: consider using cue's JSON marshaller instead of
+// "encoding/json" ... it might have extra functionality related
+// to the cue language.
+func MarshalToml(v interface{}) ([]byte, error) {
```

Comment:
I made this public (uppercase M in MarshalToml) to be consistent with the json and yaml versions.

IMO these should be private in the `cuecfg` package's interface. If folks agree, then I can send another PR to make them private.